lib/s3rver.js

```diff
@@ -145,6 +161,8 @@ class S3rver extends Koa {
 
       const { address, port, ...listenOptions } = this.serverOptions;
       this.httpServer = await this.listen(port, address, listenOptions);
+      stoppable(this.httpServer);
```

Comment:
I think I understand what `.stop` patched by `stoppable` is supposed to do, but I'm having trouble understanding why S3rver would benefit from it compared to using the `.close` function.

S3rver is intended for integration testing, so when you call `.close` you would probably want to perform cleanup as quickly as possible and do not care about open connections. If there are in-flight requests or Keep-Alive connections still open when you call `.close`, you probably forgot to wait for your requests to end before cleaning up your tests, and you would want to receive errors about prematurely closed connections rather than have these connections silently cleaned up on close.

If you are running the process independently in the background or in a container, prematurely closed connections would indicate that you closed your server earlier than your tests completed, which IMO is fine and should be expected. It would be a little unexpected for that server to remain alive while my tests finish up their current requests, only to reject future requests and fail the remainder of the tests.

So I think there are no use cases where `.stop`'s graceful functionality would be useful, and would argue against introducing `stoppable` for this reason. However if you can suggest a use case where graceful shutdown is needed I may reverse my opinion on this.